docs/pool-manager.md

```diff
@@ -0,0 +1,83 @@
+# What is the Pool Manager and what does it do?
+
+👉 A component that provisions and manages vLLM pods within the Dual-Pod (DP) Controller architecture. Can be a Kubernetes native service that maintains a pool of idle pods running just the launcher, pre-labeled for IGW’s InferencePool, and transitions them through some different lifecycle states (idle → active → sleeping → deleted) to ensure fast activation of server-running vLLM pods while minimizing cold-start latency.
```

Comment:
> pre-labeled for IGW’s InferencePool

A launcher with no awake vLLM instance has no InstancePool that it should be labeled for.